chia/wallet/nft_wallet/nft_puzzles.py

```diff
@@ -70,57 +126,101 @@ def get_nft_id_from_puzzle(puzzle: Program) -> Optional[bytes32]:
     return None
 
 
+def update_metadata(metadata, solution):
```

Comment:
```suggestion
def update_metadata(metadata: List[something], solution: Program) -> List[something]:
```
(I'm not sure what the metadata/new_metadata types are exactly)